testem.dist.js

```diff
@@ -72,6 +72,24 @@ module.exports = {
       ],
       protocol: 'browser',
     },
+    BS_Chrome_FortyOne: {
```

Comment:
Can we call this `googlebot`, and link to that web page inline?

Reply:
Definitely.  Will make this change now.